Feature comparison

neynah opened the pull request:
Addresses #221 

Changes in this commit:
- added simplified feature comparisons; emphasized hosting, administering, and support
- added graphic for Sandstorm on Oasis
- added check & x-marks
- added tagline: "Run Sandstorm anywhere." (Open to suggestions) 
- removed Oasis section
- consolidated Oasis plans into one of the 3 product options
- changed description copy for each of the products
- changed Standard to Sandstorm Standard
- changed bottom "Questions?" call to action to "Let us help you get the most out of Sandstorm"
- colour-themed each of the products (bg colour and buttons)
- change /business so that "Core and Security" will say "Sandstorm Standard" and "Business" will say "Sandstorm for Work".
Note: I recognize that "Sandstorm for Enterprise" will be under "Sandstorm for Work" and I'm OK with this for now.
- on /features removed "Choose your host" section
- update footer links
- make sure it's mobile-friendly
- changed all /get# links to /get
- mentioned SAML
- get top coloured bg sections of each plan to maintain the same height as the others

![image](https://cloud.githubusercontent.com/assets/855341/18735984/2356f3ca-8037-11e6-8d90-7c30926e9dbc.png)

![image](https://cloud.githubusercontent.com/assets/855341/18730752/d402573a-800c-11e6-8bbe-1d816feeecd3.png)

![image](https://cloud.githubusercontent.com/assets/855341/18730450/3af9759c-800b-11e6-9978-b5823357b645.png)

Screenshot from before:

![image](https://cloud.githubusercontent.com/assets/855341/18728013/34325b9e-8000-11e6-8b42-ad7bd11aa491.png)

kpreid commented:
The first screenshot contains the phrases “Sandstorm-hosted” and “Administered by Sandstorm”. All the other text consistently uses “Sandstorm.io” to refer to the company. I think this is an important distinction to keep straight particularly since the feature comparison (and Sandstorm as a whole) is significantly __about that distinction__.

neynah commented:
>The first screenshot contains the phrases “Sandstorm-hosted” and “Administered by Sandstorm”. All the other text consistently uses “Sandstorm.io” to refer to the company. I think this is an important distinction to keep straight particularly since the feature comparison (and Sandstorm as a whole) is significantly about that distinction.

I'm not 100% certain that we're consistent in all places but I agree that this consistency is important. I've updated the commit & the screenshot. Thanks!

neynah commented:
Okay, ready for review! 🌅

kentonv commented:
Hmm, I feel like we should list specific features in the checklist rather than make people click through to another page. We can also have the links, but I bet a lot of people won't click.

neynah commented:
> Hmm, I feel like we should list specific features in the checklist rather than make people click through to another page. We can also have the links, but I bet a lot of people won't click.

If we can come up with a list of 6ish items or less then this sounds good to me. I'll try to propose something.

kentonv commented:
Why 6ish? Not that I'm disagreeing, but I've seen a lot of these checklists that are a lot longer than that.

neynah commented:
> Why 6ish? Not that I'm disagreeing, but I've seen a lot of these checklists that are a lot longer than that.
I suspect that adding more than 6ish to the existing list would make the list appear too long and overbearing. I think it's a good idea to focus on the most important ones. I think it'll be easier for me to say which number I'm happy with once I figure out which features we can list.

paulproteus commented:
Feedback:

I'm looking at http://localhost:4000/get . Here are my thoughts:

- There seems to be an IMG SRC 404 fetching http://localhost:4000/images/get-oasis.svg .

- In "Self-hosted" vs "Administered by you" vs. "Administered by your IT team", I like how "by you" and "by your IT team" makes the _who_ very clear. By contrast, "Self-hosted" doesn't explain that as clearly. Therefore I suggest, "Hosted by you" or "You host on your server"

- "Hosted by you" sounds a little scary, so I would suggest adding: "Free, secure auto-updates" underneath it. Also add it to "Administered by your IT team". Or we could make it more empowering-seeming and less scary by doing s/Administered by you/You have admin privileges/" and s/Administered by your IT team/Your IT team has admin privileges".

- In "Email support" and "Priority email support", I find it difficult to perceive who is providing the email support. I suggest instead, "Free support by Sandstorm.io" and "Priority support by Sandstorm.io"

- I find it very hard to understand "$5/user/month billed annually". Doubly so since it's possible to get SfW with monthly billing. I propose instead:
    - Replace "billed annually" with "with annual discount" (but now this pricing row is two lines, eek)

- Rename "Sandstorm on Oasis" to "Hosted solution". That IMHO would be clearer. Low priority, though.

- Change "Host Sandstorm on-prem. Comes with features your that your business needs" (which is very vague language) with "Get organization-wide access control and enterprise login integration."

- The collaboration mode of Sandstorm on Oasis is very different than the collaboration mode of Sandstorm for Work, in that there is a "group" of users (your "organization") in SfW but not on Oasis. It'd be nice if we can make that distinction clear somehow. Maybe we can't in this revision, though, and maybe that's life.

- I agree with Kenton that it's good to show the specific features one gets with SfW. SfW features IMHO are:
    - Organization-wide access control
    - ActiveDirectory/LDAP/SAML integration
    - Branding customization

- I would put the SfW one in the middle, and make it bigger. See e.g. the Oasis pricing, as I recall.

Further feedback coming soon. Thanks for your patience. Feel free to ship this as-is IMHO. Life is short.

kentonv commented:
Instead of "email support" for the free version, should we say "community support"? That is: you can ask questions on sandstorm-dev, IRC, etc. It's true that we generally answer questions to support@sandstorm.io from free users, but I don't want to promise this to non-paying users since there will come a time when it becomes too much for us to handle.

neynah commented:
Thanks for the feedback everyone. @paulproteus I made a handful of the changes you suggested and decided to not make the other changes since I'm not convinced they would be improvements. I'm happy to discuss those if you like.

Additional changes completed:
- Changed "Sandstorm for Work" description to say "Organization-wide access control and login integration for your business" instead of "Host Sandstorm on-prem. Comes with features and integrations your business needs."
- Changed "Email support" to "Community support". I'm not 100% satisfied with "Community support" since it almost sounds like we're not offering support at all but I have no better suggestions at the moment. Maybe "Forum & IRC support"?
- Added feature lists for each section; I'm not certain about my word choices for: "Individual app sandboxing", "Multi-app search", "App notifications", and "Personalization & whitelabeling". Perhaps people have better suggestions!
- Organized features by "Sandstorm Standard features" and "Sandstorm for Work features"
- Moved call-to-actions up since the feature list pushed them too far down the page
- Designed a 2nd checkmark so there's a light and dark one now
- Fixed the get-oasis.svg 404 by including the image in the PR :)

I'd love to hear string suggestions by the end of today so we can get this shipped ASAP.

![image](https://cloud.githubusercontent.com/assets/855341/18803911/a08ff012-81a9-11e6-9354-3921e27b2234.png)

ocdtrekkie commented:
I would maybe suggest that "Community support" is kinda expected for an open source/self-hosted solution, but that perhaps, while not "priority", paying Oasis customers will probably expect "Email support"

kentonv commented:
@ocdtrekkie Yes, I meant "community support" for non-paying self-hosted users. We should probably say "email support" for Oasis.

kentonv commented:
I changed Oasis to "email support" in 048725aa9015381a24cb3750b79c935634e50878.

I found that the "get" buttons were overlapping with the "Sandstorm Standard features" heading on IE. Fixed with b62c1504f1828a610c6530b190abe3571b5e3c24.
